libs/ui-toolkit/src/components/tabs/tabs.tsx

```diff
@@ -2,22 +2,48 @@ import * as TabsPrimitive from '@radix-ui/react-tabs';
 import classNames from 'classnames';
 import type { ReactElement, ReactNode } from 'react';
 import { Children, isValidElement, useState } from 'react';
+import { useLocalStorage } from '@vegaprotocol/react-helpers';
 
 interface TabsProps {
   children: ReactElement<TabProps>[];
   active?: string;
+  persistId?: string;
 }
 
-export const Tabs = ({ children, active: activeDefaultId }: TabsProps) => {
+export const Tabs = ({
+  children,
+  active: activeDefaultId,
+  persistId: persistIdLocalStorageKey,
```

Comment:
suggestion (non blocking): For me it shouldn't be the responsibility of the tab component (a simple presentational component) to handle local storage/persistance, and this should be lifted to the page component. From there we'd just pass down the active tab as a prop.